android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerButtonViewManager.kt

```diff
@@ -224,18 +233,12 @@ class RNGestureHandlerButtonViewManager : ViewGroupManager<ButtonViewGroup>(), R
         foreground = selectable
         if (_backgroundColor != Color.TRANSPARENT) {
           setBackgroundColor(_backgroundColor)
+          updateBackgroundColor(_backgroundColor, borderRadius, selectable)
```

Comment:
```suggestion
          updateBackgroundColor(_backgroundColor, borderRadius, null)
```

Now the ripple effect is applied to both the background and the foreground (you can check by comparing two buttons side-by-side, the one with `foreground={true}` is darker when pressed). Since the foreground is already set to the selectable drawable, there's no need to also add it to the background.

Reply:
Good point, I've just checked that and it indeed was darker. 

Done in [ebcf766](https://github.com/software-mansion/react-native-gesture-handler/pull/2642/commits/ebcf766d4fbedbfabec8b64f849fa938bc24fb37)